link/src/main/java/com/stripe/android/link/injection/LinkInjector.kt

```diff
@@ -0,0 +1,8 @@
+package com.stripe.android.link.injection
+
+import com.stripe.android.core.injection.Injector
+
+/**
+ * Mark an [Injector] that can inject into [LinkInjectable] classes.
+ */
+internal interface LinkInjector : Injector
```

Comment:
Probably rename it to a more generic name like `NonFallbackInjector` so that we could move them to `stripe-core` later

Reply:
Done